README.md

```diff
@@ -483,6 +484,52 @@ The data format for donut, pie, and radial bar are the simplest. They are
 just any single value of type Numeric.
 
 
+## Options (Reference)
+ApexCharts.rb supports [all options from apexcharts.js](https://apexcharts.com/docs/options/), but instead of camelCase, you have to write them in snake_case.
```

Comment:
You can write it in snake_case, but you don't have to.
Please change "have to" to "can".